src/client/components/StatusCard/status-card.component.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Card from '../card/card.component';
+import Card from '../Card/card.component';
```

Comment:
Why are you changing this? The folder is lower case. Please make sure that files and paths are lower case throughout.

Reply:
Sorry zaki, i got it confused with the Footer and StatusCard. I am now changing everything and conform with the configs set.

Reply:
They have different casing than the one on the card that is why I change it.